src/SFA.DAS.AssessorService.Web/Views/Application/Standard/StandardVersion.cshtml

```diff
@@ -65,12 +65,19 @@
                                 <a class="govuk-link" href="@Url.Action("SequenceSignPost", "Application", new { Id = version.ApplicationId })">View</a>
                             </td>
                         }
+                        else if (version.VersionStatus == VersionStatus.FeedbackAdded)
```

Comment:
Possibly missed in a previous PR, the title of this page is "Confirm Standard" which I don't think is what this page is doing?

Reply:
Fixed